torch/_torch_docs.py

```diff
@@ -4272,6 +4272,59 @@ def merge_dicts(*dicts):
     {out}
 """.format(**common_args))
 
+add_docstr(torch.xlogy,
+           r"""
+xlogy(x, y) -> Tensor
+
+Compute x*log(y) so that the result is 0 if x = 0.
+Similar to SciPy's `scipy.special.xlogy`.
+
+.. math::
+    \text{out}_i = x_i * \log{{y_i}}
+
+Args:
+    x (tensor): Multiplier
```

Comment:
I suggest not bothering with giving these descriptions. This can just say:

```
self (Tensor)
other (Number or Tensor)
```

Note that both Number and Tensor are capitalized in this context.